### DISCUSSION TITLE
Memory leak from CVE-2022-48541

### DISCUSSION THREAD

**Discussion by mikuamuro** [ImageMagick/ImageMagick#6805] [Help]
### ImageMagick version

6.9.11

### Operating system, version and so on

Ubuntu

### Description

Hi,
I am just wondering if there will be a patch to the version from the Ubuntu universe repository for this CVE.

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-48541
A memory leak in ImageMagick 7.0.10-45 and 6.9.11-22 allows remote attackers to perform a denial of service via the "identify -help" command.

Thanks,
Miki

**Comment by dlemstra** [comment]
We don't know. We are not responsible for the Ubuntu distribution.

**Comment by urban-warrior** [comment]
The Linux distro teams typically track patches we apply to repair vulnerabilities and apply them to their internal distributions  of ImageMagick.  However, whether this particular CVE was patched, is unknown to us.
